src/sonic-yang-models/yang-models/sonic-port.yang

```diff
@@ -13,6 +13,10 @@ module sonic-port{
 		prefix ext;
 	}
 
+    import sonic-device_metadata {
+        prefix sdm;
+    }
```

Comment:
>     [](http://example.com/codeflow?start=0&length=4)

nit: Use /t since all other yang files are using /t to indent?

Reply:
Fixed in latest commit